### ISSUE TITLE
Add ability to "fork" repos

### ISSUE THREAD

**Issue by magik6k** [CIDgravity/gw#45]
Even for just debugging it would be really cool to have an option of copying a RIBS repo in a way that can be started and continued somewhere else

Some things will need addressing:
* Which instance is responsible for replicating deals?
  * Initially just tell the forked instance to not maintain deals for the new group
  * How does the forked instance get to know about new deals replacing expired ones?
* How do deal with data in staging storage, but with no deals yet?
  * Basically the same problem as "how to get to know about new deals" above
